### DISCUSSION TITLE
Failed image build for Ubuntu2204

### DISCUSSION THREAD

**Discussion by RobBowman** [actions/virtual-environments#7943] [General questions]
Hi, I just started using this project yesterday. I was able to build the image for UbuntuMinimal ok. However, I just tried a build for Ubuntu2204 and got the following error:

```
 azure-arm.build_vhd: Error — Either HTTP response code for 'https://chromedriver.storage.googleapis.com/<?xml' is wrong - '' or exit code is not 0 - '1'. Waiting 30 seconds before the next attempt, 1 attempts left
==> azure-arm.build_vhd: /imagegeneration/helpers/install.sh: line 29: ?xml: No such file or directory
==> azure-arm.build_vhd: /imagegeneration/helpers/install.sh: line 32: [: -eq: unary operator expected
    azure-arm.build_vhd: Error — Either HTTP response code for 'https://chromedriver.storage.googleapis.com/
```

Anyone else seen this?

**Comment by vpolikarpov-akvelon** [comment]
Hi @RobBowman. Google Chrome team have changed the way they serve Chrome Driver recently so it's required to adjust build process accordingly. We are working on this, here is an issue to track activity: https://github.com/actions/runner-images/issues/7933

By the way, Ubuntu image generation have already been fixed.
